gum/gumdefs.h

```diff
@@ -99,7 +99,7 @@ enum _GumBranchHint
 
 struct _GumCpuContext
 {
-#if !defined (__arm__) && !defined (__aarch64__)
+#if defined (HAVE_I386)
```

Comment:
We unfortunately cannot use any defines from `config.h` as is only available while building frida-gum, and this header is installed by `make install`.